client/qiskit_serverless/core/client.py

```diff
@@ -191,4 +190,9 @@ def list(self, **kwargs) -> List[RunnableQiskitFunction]:
 
     def widget(self):
         """Widget for information about provider and jobs."""
+        # prevent ciclic import
+        from qiskit_serverless.visualizaiton import (  # pylint: disable=import-outside-toplevel
```

Comment:
@korgan00 can we fix the typo `visualizaiton` to `visualization`? The name of the folder is also wrong. Thanks in advance 🙏